pom.xml

```diff
@@ -1053,6 +1053,7 @@
                     <artifactId>maven-deploy-plugin</artifactId>
                     <version>3.1.4</version>
                     <configuration>
+                        <deployAtEnd>true</deployAtEnd>
```

Comment:
This is also the default for the `central-publishing-maven-plugin`; just an extra safety net to increase the likelihood of an all-or-nothing deployment.